src/test/java/com/google/edith/UserInsightsServiceTest.java

```diff
@@ -254,13 +303,36 @@ public void createJson_withUnkownUser_returnsDefaultJson() {
    * Assigns the parameters to the given entity.
    *
    * @param entity the entity to update
+   * @param name the name of the item
    * @param price price
    * @param quantity quantity
    * @param date date
    */
-  private void setEntityProperties(Entity entity, double price, int quantity, String date) {
+  private void setEntityProperties(
+      Entity entity,
+      String name,
+      String userId,
+      String category,
+      double price,
+      int quantity,
+      String date) {
+    entity.setProperty("name", name);
+    entity.setProperty("userId", userId);
+    entity.setProperty("category", category);
     entity.setProperty("price", price);
     entity.setProperty("quantity", quantity);
     entity.setProperty("date", date);
   }
+
+  private boolean compareOrderedLists(List<WeekInfo> expected, List<WeekInfo> actual) {
+    if (expected.size() != actual.size()) {
+      return false;
+    }
+    for (int i = 0; i < expected.size(); i++) {
+      if (!expected.get(i).equals(actual.get(i))) {
+        return false;
+      }
+    }
+    return true;
+  }
```

Comment:
Does assertEquals not check order? I forget tbh. Was hoping there was a Junit method to do this, but didn't find any

Reply:
No, it checks memory locations.